<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Lounge</class>
 <widget class="QWidget" name="Form_Lounge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="BACKGROUND">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_title_gameID">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_gameID">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>182</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 32pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="Widget_PlayerList" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>351</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Widget_PlayerList{
	border: 1px solid black;
	border-radius: 10px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>9</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <spacer name="VSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>311</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="Widget_BTN" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#Widget_BTN{
	border: 1px solid black;
	border-radius: 10px;
}</string>
   </property>
   <widget class="QPushButton" name="btn_leave">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>离开房间</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ready">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>准备游戏</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="HLINE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>169</y>
     <width>211</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HLINE_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>241</y>
     <width>211</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
